src/jetpack-editor-setup.js

```diff
@@ -85,9 +85,16 @@ export function registerJetpackBlocks( { capabilities } ) {
 		capabilities.tiledGalleryBlock,
 		'jetpack/tiled-gallery'
 	);
+	hideBlockByCapability(
+		__DEV__ && capabilities.videoPressBlock,
```

Comment:
Adding the `__DEV__` flag here (vs. as part of the `supportedJetpackBlocks` object, [like the Tiled Gallery block](https://github.com/wordpress-mobile/gutenberg-mobile/blob/e4890c9a67dca19758aa92e6c18912dc0187d428/src/jetpack-editor-setup.js#L30-L32)) because the VideoPress block lives in a separate package and is never passed to [this function](https://github.com/Automattic/jetpack/blob/b6c867d80387206e11eb9f8bb9d987ebc3b6187c/projects/plugins/jetpack/extensions/shared/register-jetpack-block.native.js#L22) for checking availability.

Reply:
Good catch! I checked this change using the iOS installable build and although I can't insert the VideoPress block, it's still being rendered. The main reason for this is that `hideBlockByCapability` function hides the block but doesn't prevent being registered, hence being rendered as a regular block instead of unsupported.

As a workaround, we could implement a native version of `videopress/src/client/block-editor/blocks/video/index.ts` file ([where the block is registered](https://github.com/Automattic/jetpack/blob/2c1ee9cbd75f34375c1cb93a23dab9654e0bebdd/projects/packages/videopress/src/client/block-editor/blocks/video/index.ts#L19-L30)), and either:
1. Expose a `registerVideoPressBlock` function that registers the block in order to be used in Gutenberg Mobile.
2. Add the needed checks to conditionally register the block.

Reply:
Ah, thank you for spotting that, I'll work on a `registerVideoPressBlock` function.